telemetry/definitions/commonDefinitions.json

```diff
@@ -901,11 +901,20 @@
             "type": "double",
             "description": "Captures the number of files generated as a part of code generation iteration"
         },
-
         {
             "name": "amazonqRepositorySize",
             "type": "double",
             "description": "Captures  size of the source code"
+        },
+        {
+            "name": "amazonqNumberOfReferences",
+            "type": "double",
+            "description": "Captures the number of references"
+        },
+        {
+            "name": "amazonqEndOfTheConversationLatency",
+            "type": "double",
+            "description": "The time it takes to end amazonq conversation"
```

Comment:
This is technically the entire duration of the conversation right? Can we say something like the total time from start to end

Reply:
renamed